cpp/ps0006/ps150days.cpp

```diff
@@ -109,7 +99,7 @@ int main()
     cout << "===============" << endl;
 
     counter(200, -44); // Massive number of iterations for our humble counter
-}
-
-
 
+    return 0;
+}
+// great work 😎 10/10

```

Comment:
- [x] program is in the correct path
- [x] no syntax errors while compilation
- [ ] no runtime errors, bugs or warnings while execution
- [x] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code
- [x] used `for` loop in a right way to to count up and down
+0.5 bonus mark for your effort ⭐

you can check the [problem solution](https://onlinegdb.com/vdj-VXYv7) for more